Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3619,7 +3619,7 @@ public void run() {
 	 * @see BranchError	
 	 */
 	public interface BranchListResponseListener {
-		public void onReceivingResponse(JSONArray list, BranchError error);
+		void onReceivingResponse(JSONArray list, BranchError error);
```

Comment:
Oh gawd I am sorry :person_frowning:. I am the Android Lead at your client Evergive, at Sherman Ave.  I would have thought @aaustin moved everyone by now. Oh well!

That said, I don't think anyone inspects the lint results, because it pointed out most of the cleanup edits. Or the Android guy(s)?  @sojanpr ? just don't care about it.